src/hotspot/share/jfr/jfr.hpp

```diff
@@ -28,6 +28,11 @@
 #include "jni.h"
 #include "memory/allocation.hpp"
 
+#include "jfr/support/jfrIntrinsics.hpp"
```

Comment:
Don't think we need to include jfrIntrinsics.hpp (not even in .cpp) as we can declare this entry point unconditionally.

Reply:
removed, thanks.